shared/settings/feedback/index.tsx

```diff
@@ -31,12 +32,11 @@ type State = {
 }
 
 class Feedback extends React.Component<Props, State> {
-  state = {
-    email: null,
-    feedback: '',
-    sendLogs: true,
-    showSuccessBanner: false,
+  constructor(props) {
+    super(props)
+    this.state = {email: null, feedback: this.props.feedback, sendLogs: true, showSuccessBanner: false}
```

Comment:
FWIW you could also do this and get rid of the constructor itself:

```
class Feedback ... {
  state = {
    ...
  }
}
```

Reply:
Can I? I'm using `this.props.feedback` here so I think I can't.

Reply:
Yup; we have a bunch of them:

```
%   rg 'state = \{.*this\.props' | head -n1
devices/index.tsx:  state = {revokedExpanded: this.props._stateOverride ? this.props._stateOverride.revokedExpanded : false}
```

Reply:
(You don't have to change this though; just wanted to make sure you know that!)

Reply:
Cool! I think your way is cleaner then.